src/components/presentational/TimeSeriesChart/TimeSeriesChart.tsx

```diff
@@ -129,10 +159,36 @@ export default function TimeSeriesChart(props: TimeSeriesChartProps) {
                 } else if (domainMin !== undefined) {
                     domain = [domainMin, "auto"];
                 }
```

Comment:
Is there an option for `domain = ["auto", domainMax]` ?

Reply:
@brandoncharnesky the domainMax is only needed for setting the gradient. The graph will detect the max value.